.devcontainer/devcontainer.json

```diff
@@ -70,7 +76,7 @@
                 "git.inputValidationSubjectLength": 72,
                 // If clangd was obtained from a package manager, its path can be set here.
                 // Note: This has to be adjusted manually, to the "llvm_version" from above
-                "clangd.path": "clangd-19",
+                "clangd.path": "clangd-20",
                 "clangd.arguments": [
                     "--header-insertion=never" // See https://github.com/clangd/clangd/issues/1247
```

Comment:
oh hey. with clangd-20, we should be able to add config for system header insertion now that this issue is closed. I'll make an issue. (i.e. no change requested here).

Reply:
https://github.com/LadybirdBrowser/ladybird/issues/5980